src/main/java/com/movie_collection/HelloApplication.java

```diff
@@ -1,16 +1,13 @@
 package com.movie_collection;
```

Comment:
yeah I have done the main in new pr and since it does nothing that cannot be done after I would not push this

Reply:
https://github.com/TomassSimko/Private-Movie-Collection/pull/7/commits/c327b4d35a561959bc104c192001071ff3d61114